Symfony/CS/Fixer/Symfony/SingleQuoteFixer.php

```diff
@@ -35,12 +35,13 @@ public function fix(\SplFileInfo $file, $content)
             if (
                 '"' === $content[0] &&
                 false === strpos($content, "'") &&
-                // regex: odd number of backslashes, not followed by double quote
-                !preg_match('/(?<!\\\\)(?:\\\\{2})*\\\\(?!["\\\\])/', $content, $m)
+                // regex: odd number of backslashes, not followed by dollar or double quote
+                !preg_match('/(?<!\\\\)(?:\\\\{2})*\\\\(?![("\$)\\\\])/', $content, $m)
```

Comment:
This is still wrong. You have to remove the `(` and `)`.

At the moment the regex is this: 
"odd number of backslashes, not followed by **parentheses**, dollar or double quote"